daq/runner.py

```diff
@@ -884,9 +885,13 @@ def _send_device_result(self, mac, test_results):
 
     def _calculate_device_result(self, test_results):
         for module_result in test_results.get('modules', {}).values():
+            if report.ResultType.EXCEPTION in module_result:
```

Comment:
It should also check for non zero ResultType.RETURN_CODE

Reply:
Done